atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -165,7 +165,9 @@ public Builder dynamicColumns(List<NameComponentDescription> components, ValueTy
         builder.setSweepStrategy(getSweepStrategy());
         builder.setAppendHeavyAndReadLight(isAppendHeavyAndReadLight());
         builder.setNameLogSafety(getNameLogSafety());
-        builder.setDenselyAccessedWideRows(hasDenselyAccessedWideRows());
+        if (hasDenselyAccessedWideRows()) {
+            builder.setDenselyAccessedWideRows(hasDenselyAccessedWideRows());
```

Comment:
out of curiosity, are you going to make the change to compare objects instead of byte arrays at a future point in time?

Reply:
Discussed offline with @gmaretic, I think we'll do it now actually

Reply:
Turns out this is not as trivial as I thought because of an edge case where if you want to write generic table metadata, you won't. I'll revert to the old method for now.